posts/2021-11-30-microprofile-5-22001-beta.adoc

```diff
@@ -0,0 +1,203 @@
+</featureManager>
+----
+
+Alternatively, you can enable the individual MicroProfile features that you need, for example:
+
+[source, xml]
+----
+<featureManager>              
+    <feature>mpOpenTracing-3.0</feature>
+</featureManager>
+----
+
+For more information about this update check out the link:https://github.com/eclipse/microprofile/releases/tag/5.0[MicroProfile 5.0 Release] on GitHub.
+
+
+[#rest_client]
+=== MicroProfile Rest Client 3.0
+
+MicroProfile Rest Client is an API intended to enable developers to write type-safe interfaces to abstract and invoke remote RESTful services. This is the 3.0 release of MicroProfile Rest Client, and it adds support for Jakarta EE 9.1 technologies - from a developer's perspective, the only change from the previous release (2.0) is the package space name change of Jakarta packages from `javax.*` to `jakarta.*`. Another change is that the Open Liberty implementation has changed from Apache CXF to RESTEasy - this change brings with it some behavior and property changes (most of which are already documented as link:{url-prefix}/docs/latest/reference/jaxrs-dif.html[differences between `jaxrs-2.1` and `restfulWS-3.0`]).  
+
+To use this new feature, you would need to add `mpRestClient-3.0` to the `featureManager` element in the server.xml. The code should be similar to previous versions of MP Rest Client, but the packages should change from `javax.*` to `jakarta.*`.
+
+[source, xml]
+----
+<featureManager>              
+    <feature>mpRestClient-3.0</feature>
+</featureManager>
+----
+
+For more information, check out:
+
+* link:https://github.com/eclipse/microprofile-rest-client[Eclipse MicroProfile Rest Client GitHub]
+* link:http://download.eclipse.org/microprofile/microprofile-rest-client-3.0-RC5/microprofile-rest-client-spec-3.0-RC5.html[MicroProfile Rest Client 3.0 Specification]
+* link:http://download.eclipse.org/microprofile/microprofile-rest-client-3.0-RC5/apidocs/[MicroProfile Rest Client 3.0 API Javadoc]
+
+
+[#jpa_persistence]
+=== Define JPA persistence properties at server scope
+
+This new JPA enhancement adds the ability to declare "default" JPA persistence properties, to all container-managed persistence contexts, as a Liberty server.xml configuration.
+
+Previously, if a persistence property needed to be set for all persistence.xml configuration files, you would manually update all persistence.xml files in all applications. This could end up requiring hundreds of manual updates and/or rebuilding of applications.
+With this enhancement, you can specify persistence properties, in the server.xml, that will propagate to all container-managed persistence units for applications installed on that server.
+
+In order to start using the new feature, you only need to add the new `<defaultProperties>` configuration to your `<jpa>` configuration, in server.xml, along with the persistence properties that you want to apply to all container-managed persistence units:
```

Comment:
To start using the new feature, add the `<defaultProperties>` configuration element to the `<jpa>` configuration in your `server.xml` file. Specify the persistence properties that you want to apply to all container-managed persistence units, as shown in the following example: